python/lsst/validate/drp/checkAstrometryPhotometry.py

```diff
@@ -50,8 +54,8 @@ def getCcdKeyName(dataid):
 
     Notes
     -----
-    Motiviation: Different camera mappings use different keys to indicate 
-      the different amps/ccds in the same exposure.  This function looks 
+    Motiviation: Different camera mappings use different keys to indicate
+      the different amps/ccds in the same exposure.  This function looks
```

Comment:
I totally agree.  This sort of thing needs to be handled per-camera (in the mapper?) if we can't agree on a single naming convention.   I think KT dislikes the idea of two names for one thing, but it would be a viable solution to support a single name for e.g. a sensor and provide aliases.